src/index.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The index file for sending pending-review notifications."""
```

Comment:
Why is this called index.py -- in what way is it an "index"? Why not main.py or send_pending_review_notifications.py or similar?

Reply:
I have changed the file name to main for now but I am also open for suggestions.